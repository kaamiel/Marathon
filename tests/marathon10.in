addUser 0 10
addUser 10 11
addUser 11 12
addUser 0 20
addUser 20 21
addUser 21 22
addUser 0 30
addUser 30 31
addUser 31 32
addMovie 10 10
addMovie 11 12
addMovie 12 11
addMovie 20 12
addMovie 21 10
addMovie 22 11
addMovie 30 10
addMovie 31 11
addMovie 32 12
marathon 10 2
marathon 10 3
marathon 20 1
marathon 20 2
marathon 30 3
marathon 30 4
