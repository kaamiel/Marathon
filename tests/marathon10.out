OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
OK
12 10
12 10
12
12
12 11 10
12 11 10
